great_expectations/data_context/util.py

```diff
@@ -181,8 +181,19 @@ def mask_db_url(cls, url: str, use_urlparse: bool = False, **kwargs) -> str:
         Returns:
             url with password masked e.g. "postgresql+psycopg2://username:***@host:65432/database"
         """  # noqa: E501
+
+        from great_expectations.datasource.fluent.config_str import ConfigStr
+
+        try:
+            ConfigStr.validate_template_str_format(url)
```

Comment:
Can we call `ConfigStr.str_contains_config_template` instead? That seems to do the same thing as `validate_template_str_format` except returns a bool and doesn't raise. You could then remove the `try/except` here.

Reply:
Oh yeah, that's much better. Thanks!